ocs_ci/utility/service.py

```diff
@@ -48,10 +53,19 @@ def verify_service(self, node, action):
         """
         nodeip = self.nodes[node.name]
         result = exec_cmd(
-            f"ssh core@{nodeip} sudo systemctl is-active {self.service_name}.service",
+            f'ssh -o "StrictHostKeyChecking no" -J root@{self.bastion_ip} core@{nodeip} '
```

Comment:
Luke, I looked into using oc debug. However, it will work only when you shutdown kublet.service. To start it back, oc debug will not work as the internal link between oc command and nodes is broken due to kubelet service being down. This is the same issue we face with inband/out-of-band access to nodes.

Reply:
ok

Reply:
Is there a requirement to perform stop by itself?   You could replace the systemctl stop start sequence with systemctl restart.   This should reboot the kubelet and would not be dependent on the network.

Reply:
Luke, there are separate step requirements for stop and start and restart from ocs-ci perspective.